app.js

```diff
@@ -22,4 +22,23 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', {beersFromApi});
+      console.log(beersFromApi);
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res, next) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      res.render('random-beer', { responseFromAPI });
```

Comment:
Yes! what you say in your comment is another way of getting the data, in which you extract the random beer from the array and you send it as an object. Though if you want to use partials with this it might give you issues if you go with this approach.